pkg/otlp/metrics/metrics_translator_test.go

```diff
@@ -268,37 +268,73 @@ func seconds(i int) pcommon.Timestamp {
 }
 
 var exampleDims = newDims("metric.example")
+var rateAsGaugeDims = newDims("placeholder.rate.as.gauge.metric")
 
 func TestMapIntMonotonicMetrics(t *testing.T) {
-	// Create list of values
-	deltas := []int64{1, 2, 200, 3, 7, 0}
-	cumulative := make([]int64, len(deltas)+1)
-	cumulative[0] = 0
-	for i := 1; i < len(cumulative); i++ {
-		cumulative[i] = cumulative[i-1] + deltas[i-1]
-	}
+	t.Run("diff", func(t *testing.T) {
+		// Create list of values
+		deltas := []int64{1, 2, 200, 3, 7, 0}
```

Comment:
You can reuse the code defining the point for both subtests (create a helper function to build a new point for you)

Reply:
Good callout, I added a helper. Let me know if it looks good ?